### ISSUE TITLE
SMS messaging throws JSON.parse error

### ISSUE THREAD

**Issue by homecarecormac** [twilio/twilio-node#715]
### Issue Summary
Sending SMS threw error `Unexpected token < in JSON at position 0`.
This doesn't happen always.
This happened only sometimes.
Not sure what makes it happening from time to time.

### Steps to Reproduce
1. Install `twiliow` npm package
2. Configure proper `accountSid` and `authToken`
3. Execute following code snippet

### Code Snippet
```node
const client = require('twilio')(accountSid, authToken);
await client.messages.create({
  body: messageBody,
  to: receipient,
  from: sender,
});
```

### Exception/Log
```
SyntaxError: Unexpected token < in JSON at position 0
```

### Technical details:
* twilio-node version: 3.58.0
* node version: 14

**Comment by mariomui** [comment]
@childish-sambino yeah, i submitted a PR fix for this.
https://github.com/twilio/twilio-node/pull/852/commits/1b4eb8bd05a0edf9d55825d926e6777f6ff4e379
my question would be does the member variables have to be populated on a bad parse? And is there a way to forward the JSON.parse error?

**Comment by childish-sambino** [comment]
Fixed by #855
